openlibrary/core/bookshelves.py

```diff
@@ -202,6 +202,97 @@ def count_total_books_logged_by_user_per_shelf(
         result = oldb.query(query, vars=data)
         return {i['bookshelf_id']: i['count'] for i in result} if result else {}
 
+    # Iterates through a list of solr docs, and for all items with a 'logged edition'
+    # it will remove an item with the matching edition key from the list, and add it to
+    # doc["editions"]["docs"]
+    def link_editions_to_works(solr_docs):
+        linked_docs: list[web.storage] = []
+        docs_dict = {}
```

Comment:
```suggestion
        edition_key_to_work_doc = {}
```

Reply:
I feel like I have to put `docs_dict` in a picture frame somewhere, to remind myself of the sorts of variables that make perfect sense to me while coding, and completely nonsensical three days later. Thanks for the suggestion!